node/bser/index.js

```diff
@@ -120,6 +120,9 @@ Accumulator.prototype.peekInt = function(size) {
 
 Accumulator.prototype.readInt = function(bytes) {
   var ival = this.peekInt(bytes);
+  if (ival instanceof Int64 && isFinite(ival.valueOf())) {
```

Comment:
when is is expected to not be finite?
In that case, the return value here can either be `number` or `Int64`
I'd rather check the `isFinite` on the consumer side of this function's return value.